docs/alphafold_transcript_coverage.md

```diff
@@ -0,0 +1,620 @@
+
+
+# Steps
+
+
+## Files downloaded
+
+MANE Select transcripts  
+
+-   MANE.GRCh38.v0.95.summary.txt.gz
+-   <https://ftp.ncbi.nlm.nih.gov/refseq/MANE/MANE_human/current/MANE.GRCh38.v0.95.summary.txt.gz>
+-   Accessed 10/11/2021
+
+AlphaFold2 Homo sapiens Structures  
+
+-   UP000005640\_9606\_HUMAN.tar
+-   <https://ftp.ebi.ac.uk/pub/databases/alphafold/UP000005640_9606_HUMAN.tar>
+-   Accessed 9/11/2021
+
+GENCODE protein-coding transcripts  
+
+-   gencode.v39.pc\_transcripts.fa
+-   <http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/latest_release/gencode.v39.pc_transcripts.fa.gz>
+-   Accessed 20/1/22
+
+GENCODE protein-coding translations  
+
+-   gencode.v39.pc\_translations.fa
+-   <http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/latest_release/gencode.v39.pc_translations.fa.gz>
+-   Accessed 20/1/22
+
+GENCODE comprehensive gene annotation (gtf format)  
+
+-   gencode.v39.annotation.gtf
+-   <https://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_39/gencode.v39.annotation.gtf.gz>
+-   Accessed 20/1/22
+
+ClinVar variant summary table  
+
+-   variant\_summary.txt.gz
+-   <https://ftp.ncbi.nlm.nih.gov/pub/clinvar/tab_delimited/variant_summary.txt.gz>
+-   Accessed 25/1/22
+
+
+## Unpack the AlphaFold2 transcripts
+
+Unzip the AlphaFold2 structures into a folder. In my case, `~/data/af2/`  
+
+    tar -xvf UP000005640_9606_HUMAN.tar
+
+
+## Get the UniProt ID&rsquo;s corresponding to AF2 structures
+
+Quick shell command to parse the names from the AF2 structure filenames.  
+
+    ls | awk -F"-" '{print $2}' | sort | awk '!a[$0]++' > uniprots.txt
+
+20,504 unique UniProt IDs to use.  
+
+
+## Get UniProt metadata and matched ENSGs
+
+To match the UniProt accessions to GENCODE, I&rsquo;m using the ENSG&rsquo;s paired with each UniProt structure in the UniProtKB idmapping table. For whatever reasons, matching GENCODE ENST&rsquo;s to UniProtKB&rsquo;s listed ENST&rsquo;s directly is very lossy (even disregarding transcript version numbers). Here we&rsquo;re just getting the idmapping table, filtering to UniProts where we have an AlphaFold2 structure and splitting out the ENSG from `ENSG000001234; ENSG000001235` into separate rows.  
```

Comment:
```suggestion
To match the UniProt accessions to GENCODE, I'm using the ENSGs paired with each UniProt structure in the UniProtKB idmapping table. For whatever reasons, matching GENCODE Ensembl transcripts (ENSTs) to UniProtKB's listed ENSTs directly is very lossy (even disregarding transcript version numbers). Here we're just getting the idmapping table, filtering to UniProts where we have an AlphaFold2 structure and splitting out the ENSG from `ENSG000001234; ENSG000001235` into separate rows.  
```

Reply:
I think there's a number of these "&rsquo;" 's in the document resulting from exporting my notebook style doc into .md, I'll see if I can switch off this behaviour